src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/ComInterfaceGenerator.cs

```diff
@@ -395,14 +239,15 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
                         IdentifierName("InterfaceInformation"),
                         IdentifierName("InterfaceImplementation")));
 
-        private static MemberDeclarationSyntax GenerateIUnknownDerivedAttributeApplication(ComInterfaceContext context)
+        private static MemberDeclarationSyntax GenerateIUnknownDerivedAttributeApplication(ComInterfaceInfo context, CancellationToken _)
```

Comment:
We can remove the `CancellationToken _` parameters throughout. No need to pass them if we aren't using them. We can always add them later.

Reply:
That makes sense. We can keep them then.